### DISCUSSION TITLE
Gradient Checkpointing

### DISCUSSION THREAD

**Discussion by brosand** [explosion/spaCy#11085] [Help: Other Questions] [answered]
Is there any way to enable gradient checkpointing for model development?

**Comment by polm** [comment, marked as answer]
Do you mean saving training state after every N iterations or something? If so, no, that's not a feature - training just saves the best and last models. This is partly related to resuming training being kind of difficult at the moment.

If you wanted to save models more frequently you should be able to do so by modifying the training loop code pretty easily.
